dask/local.py

```diff
@@ -143,12 +147,16 @@ def start_state_from_dask(dsk, cache=None, sortkey=None):
     Examples
     --------
 
-    >>> dsk = {'x': 1, 'y': 2, 'z': (inc, 'x'), 'w': (add, 'z', 'y')}  # doctest: +SKIP
-    >>> from pprint import pprint  # doctest: +SKIP
-    >>> pprint(start_state_from_dask(dsk))  # doctest: +SKIP
+    >>> dsk = {'x': 1, 'y': 2, 'z': (inc, 'x'), 'w': (add, 'z', 'y')}
```

Comment:
This seems to be the exact same example as the one shown in the module docstring. Is that something we don't generally mind?